src/v2/view-builder/views/password/EnrollAuthenticatorPasswordView.js

```diff
@@ -11,6 +12,37 @@ const Body = BaseForm.extend({
     return loc('oie.next.button', 'login');
   },
 
+  initialize () {
+    BaseForm.prototype.initialize.apply(this, arguments);
+    const policy = this.getPasswordPolicy();
+    this.displayPasswordPolicy(policy);
+  },
+
+  displayPasswordPolicy (policy) {
+    if (policy) {
+      let listHtml = '';
+      const rulesList = getPasswordComplexityDescriptionForHtmlList( policy );
+      rulesList.forEach(rule => listHtml += `<li>${rule}</li>`);
+  
+      this.add(
+        `<section class="password-authenticator__rules">
```

Comment:
Where are the CSS naming standards coming from?

Reply:
It's BEM convention - http://getbem.com/introduction/

Reply:
Updated with new class - `password-authenticator--rules`